source/postprocess/gravity_point_values.cc

```diff
@@ -414,6 +432,47 @@ namespace aspect
             }
         }
 
+      // write quantities in the statistic file
+      const std::string name0("Stokes velocity polynomial degree");
+      statistics.add_value (name0, degree-quadrature_degree_increase);
+      statistics.set_precision (name0, 0);
+      statistics.set_scientific (name0, false);
+
+      const std::string name1("Quadrature degree increase for gravity calculation");
+      statistics.add_value (name1, quadrature_degree_increase);
+      statistics.set_precision (name1, 0);
+      statistics.set_scientific (name1, false);
+
+      const std::string name2("Average gravity (m/s^2)");
+      statistics.add_value (name2, sum_g/n_satellites);
+      statistics.set_precision (name2, 18);
+      statistics.set_scientific (name2, true);
+
+      const std::string name3("Minimum gravity (m/s^2)");
+      statistics.add_value (name3, min_g);
+      statistics.set_precision (name3, 18);
+      statistics.set_scientific (name3, true);
+
+      const std::string name4("Maximum gravity (m/s^2)");
+      statistics.add_value (name4, max_g);
+      statistics.set_precision (name4, 18);
+      statistics.set_scientific (name4, true);
+
+      const std::string name5("Average potential (m^2/s^2)");
```

Comment:
```suggestion
      const std::string name5("Average gravity potential (m^2/s^2)");
```

and the same for the two properties below.